xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -1483,7 +1483,7 @@ activitystream.event.deleteComment.rss.body=A comment has been deleted from the
 ### Deleting a page
 core.delete=Delete
 core.delete.title=Delete {0}
-core.delete.backlinksInfo=After deleting this page, the {0,choice,0#incoming links|1#incoming link|1<incoming links} will point to an empty page. To avoid this, you can select a new target.
+core.delete.backlinksMessage=After deleting this page, the {0}{1,choice,0#incoming links|1#incoming link|1<incoming links}{2} will point to an empty page. To avoid this, you can select a new target.
```

Comment:
This also won't be needed if the hint of Update Links is always visible and has the link to check the backlinks.